.github/workflows/manager_docker_images.yml

```diff
@@ -0,0 +1,87 @@
+name: Manager - Build and Push Docker image
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+    paths:
+      - 'src/manager/**'
+
+jobs:
+  release_tagged_image:
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: "src/manager"
+    permissions:
+      contents: write
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub Container Registry
+        uses: docker/login-action@v3
+        with:
+          username: marcusschiesser
+          password: ${{ secrets.DOCKER_TOKEN }}
+
+      - name: Extract tag name
+        id: tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          platforms: linux/amd64,linux/arm64/v8
+          tags: |
+            ragapp/manager:${{ steps.tag.outputs.TAG }}
+            ragapp/manager:latest
+
+      - name: Create GitHub release
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: "README.md"
+          name: Release ${{ steps.tag.outputs.TAG }}
+          bodyFile: "CHANGELOG.md"
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
don't we end up having two github releases now?

Reply:
I just updated the pipeline for tag build with pattern: `ragapp-v` or `manager-v`. Please help me to have a look.

Reply:
that's not good, we need one release `v*`